api/pom.xml

```diff
@@ -134,49 +74,67 @@
             </resource>
         </resources>
-
         <plugins>
-           <plugin>
+           <!-- Sets minimal Maven version to 3.5.4 -->
+            <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
+                <executions>
+                    <execution>
+                        <id>enforce-maven</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>3.5.4</version>
+                                </requireMavenVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            
+           <!-- Restricts the Java version to 1.8 -->
+            <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
+                    <compilerArgument>-Xlint:unchecked</compilerArgument>
                 </configuration>
             </plugin>
 
+            <!-- First sets properties for the maven-bundle-plugin and later checks if they are indeed used. -->
             <plugin>
                 <groupId>org.glassfish.build</groupId>
                 <artifactId>spec-version-maven-plugin</artifactId>
+                <version>2.0</version>
                 <configuration>
-                    <specMode>jakarta</specMode>
                     <spec>
-                        <nonFinal>${spec.non.final}</nonFinal>
-                        <jarType>api</jarType>
-                        <specVersion>${spec.version}</specVersion>
-                        <specBuild>${spec.build}</specBuild>
-                        <specImplVersion>${spec.impl.version}</specImplVersion>
-                        <apiPackage>${spec.api.package}</apiPackage>
-                        <newSpecVersion>${spec.new.spec.version}</newSpecVersion>
+                        <specVersion>3.0</specVersion>
+                        <specImplVersion>${project.version}</specImplVersion>
+                        <apiPackage>jakarta.jms</apiPackage>
                     </spec>
                 </configuration>
                 <executions>
                     <execution>
                         <goals>
                             <goal>set-spec-properties</goal>
-                            <!-- TODO:
-                            glassfish-spec-version-maven-plugin needs to be updated
-                            in order to check 'jakarta.' prefixed values in manifest entries
```

Comment:
Can you file an issue for this, and maybe poke Romain Grecourt about it?  Thanks.

Reply:
That has been done since the comment was added. So the comment could simply be removed. Checking for "jakarta." in the generated MANIFEST.MF file has long been possible.